pydatastructs/linear_data_structures/stack.py

```diff
@@ -0,0 +1,98 @@
+"""stack.py file containning defination of 'Stack' datastructure"""
```

Comment:
This can be removed.

Reply:
It is suggested as doc generator like sphinx add this to file description which help in understanding files structure when using documentation